### DISCUSSION TITLE
Very cool update!

### DISCUSSION THREAD

**Discussion by shakfu** [Wasted-Audio/hvcc#44] [General]
Hi,

Awesome! Very cool update for hvcc. It's great to revive this project which really shouldn't be neglected as it was.

It would be great to get your PR accepted at enzienaudio/hvcc and have everyone convert to a python3 version since there seem to be quite a number of projects which continue to depend on it: [rebel technology's](https://www.rebeltech.org) [uses](https://github.com/pingdynasty/hvcc) it and now [befaco](https://www.befaco.org) (with their [lich](https://www.befaco.org/lich/) eurorack module) and also electrosmith's [daisy](https://www.electro-smith.com/daisy) via [py2dsy](https://github.com/electro-smith/pd2dsy).

What else needs to be done? I just forked your project and if I get some time, I'll check it in more detail over the weekend.

**Comment by dromer** [comment]
Thnx! Indeed I really wanted to move things forward and make it future-proof.

Can sure use some help to test all the different parts, as I don't use anything other than my newly added DPF generator :)

**Comment by shakfu** [comment]
Sounds good. I can try to get some tests up and running when I free up.

Your DPF generator sure looks interesting. Haven't heard of the DISTRHO Plugin Framework before.

**Comment by dromer** [comment]
Currently only supporting Linux, but it would be nice to be able to test DPF on windows and macos as well of course :)

**Comment by shakfu** [comment]
I don't use windows any more but I can test on a Mac (Catalina)

**Comment by shakfu** [comment]
I just added a basic test (based on the c generation mode) which required  a couple of minor fixes to the test_signal test case (see my recent commits). This test basically generates a wav file and plays It. 

It works but required a few tweaks to the current test case. I have made the changes in a separate folder [tests_new](https://github.com/shakfu/hvcc/tree/develop/tests_new) to keep it separate from the main code and also updated the README slightly.

also see: https://github.com/pingdynasty/OpenWare/issues/54 which should be of interest.
